master/buildbot/status/status_gerrit.py

```diff
@@ -349,6 +350,9 @@ def sendCodeReview(self, project, revision, result):
         if message:
             command.append("--message '%s'" % message.replace("'", "\""))
 
+        if self.gerrit_notify:
+            command.append("--notify %s" % self.gerrit_notify)
```

Comment:
You sure this isnt supposed to be:
    command.extend(['--notify', self.gerrit_notify])
? Note that I know little of this step, but this is how other steps would normally work.